webui/react/src/routes/index.ts

```diff
@@ -22,14 +21,18 @@ export interface RouteConfigItem extends RouteProps {
   needAuth?: boolean;
 }
 
-export const appRoutes: RouteConfigItem[] = [
+const dashboardRoute =
   {
-    component: Determined,
-    id: 'det',
+    component: Dashboard,
+    icon: 'user',
+    id: 'dashboard',
     needAuth: true,
     path: '/det/dashboard',
-    title: 'Determined',
-  },
+    title: 'Dashboard',
+  };
+
+export const appRoutes: RouteConfigItem[] = [
+  dashboardRoute,
```

Comment:
nit: No strong opinions here but I prefer having all the routes in one list and then having a default reference one of the items of the list. If we are changing the default app route in the future to another one in the list, we can just change the index instead of shuffling route code around.

Reply:
hmm my thought was that the downside with the index approach is that if the list order changes you'll also unexpectedly change the default route

separate: by "one list" do you mean merging appRoutes and sidebarRoutes? I did this but then reverted it to keep this PR simple. but I think we could do that as well and then show anything that has an icon (or an additional new field for more flexibility) in the sidebar

Reply:
updated it to be index based